lib/modules/datasource/index.ts

```diff
@@ -388,7 +388,7 @@ function fetchCachedReleases(
   config: GetReleasesInternalConfig,
 ): Promise<ReleaseResult | null> {
   const { datasource, packageName, registryUrls } = config;
-  const cacheKey = `${cacheNamespace}${datasource}${packageName}${config.registryStrategy}${String(
+  const cacheKey = `datasource-mem-cache:releases-memoization:${datasource}:${packageName}:${config.registryStrategy}:${String(
```

Comment:
```suggestion
  const cacheKey = `datasource-memoization:${datasource}:${packageName}:${config.registryStrategy}:${String(
```

Couldn't this be shortened to this?